event_routing_backends/processors/xapi/tests/fixtures/expected/edx.certificate.created.json

```diff
@@ -0,0 +1,48 @@
+{
+   "actor": {
+      "objectType": "Agent",
+      "account": {
+        "name": "32e08e30-f8ae-4ce2-94a8-c2bfe38a70cb",
+        "homePage": "http://localhost:18000"
+      }
+   },
+   "id":"957c726c-0fd0-5b39-8730-fc9a94ba12b1",
+   "object":{
+      "id":"http://localhost:18000/certificates/07b6cc1db05346c7841e587a56de0ee0",
+      "definition":{
+         "name":{
+            "en":"Certificate course-v1:edX+DemoX+Demo_Course"
```

Comment:
There is a convention to choose this name. I mean without dash or you only selected that way?

Reply:
I'm not aware of any convention this was something that I chose but if you have a different opinion let me know anyway this pr will be opened in upstream probably they have a different opinion

Reply:
Actually many xapi expected don't have the "name" in `object. definition`. I found this one, but also only has a unit name.
So is weird.
https://github.com/nelc/event-routing-backends/blob/master/event_routing_backends/processors/xapi/tests/fixtures/expected/problem_check(server%2Cmultiple_questions%2Cpartial_correct).json#L24

I think that would not have problem and I like that at least you add the certificate instead of only the name like in the above case.

Reply:
indeed that is a thing that we had to add in filters because the statements that we need doesn't have that name and the Nelc guide ask for that